dataland-frontend/src/utils/RequestUtils.ts

```diff
@@ -111,3 +116,25 @@ export function accessStatusBadgeClass(accessStatus: AccessStatus): string {
       return 'p-badge outline rounded';
   }
 }
+
+/**
+ * Defines the color of p-badge
+ * @param priority priority of a request
+ * @returns p-badge class
+ */
+export function priorityBadgeClass(priority: RequestPriority): string {
```

Comment:
This really looks like these styles should be globally defined and used here, but since you only copied existing code, you don't have to fix it.

Reply:
then I won't :)